website/src/components/Changelog/GUI.tsx

```diff
@@ -14,6 +14,12 @@ export default function GUI({ os }: { os: OS }) {
             Configures the IPC service to log to journald.
           </ChangeItem>
         )}
+        {os === OS.Windows && (
+          <ChangeItem pull="8268">
+            Fixes a dead-lock that could occur during shutdown of the TUN
+            device if there were still packets queued for sending.
```

Comment:
```suggestion
            Fixes a hang that could occur after signing out, causing an
            <code>os error 231</code> to be shown on the next launch.
```

Reply:
Just thought this might be more relatable to the issue non-technical users are experiencing.

Reply:
Interesting. I got that code every time. It required restarting the GUI while the IPC was in the hung state.

Reply:
I've updated it to something more generic. There might be other bugs that cause us to fail to connect to the named pipe so stating this as the fix for that isn't quite correct.